spark/src/main/scala/geotrellis/spark/io/accumulo/AccumuloLayerDeleter.scala

```diff
@@ -0,0 +1,41 @@
+package geotrellis.spark.io.accumulo
+
+import geotrellis.spark.LayerId
+import geotrellis.spark.io._
+import org.apache.accumulo.core.client.{BatchWriterConfig, Connector}
+import org.apache.accumulo.core.security.Authorizations
+import spray.json.JsonFormat
+import org.apache.accumulo.core.data.{Range => AccumuloRange}
+import spray.json.DefaultJsonProtocol._
+import scala.collection.JavaConversions._
+
+class AccumuloLayerDeleter(attributeStore: AttributeStore[JsonFormat], connector: Connector) extends LayerDeleter[LayerId] {
+
+  def delete(id: LayerId): Unit = {
+    if (!attributeStore.layerExists(id)) throw new LayerNotFoundError(id)
+    val (header, _, _, _, _) = try {
```

Comment:
Is there an exception that we can catch here that would be thrown if the layer doesn't exist? It's not ideal to make a listing call and then a fetch call, just because each listing costs $$ so if we can get away with not making it it would be the cheaper option